kola/tests/misc/tang.go

```diff
@@ -0,0 +1,300 @@
+package misc
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"net"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+
+	tutil "github.com/flatcar/mantle/kola/tests/util"
+
+	"io/fs"
+
+	"github.com/anatol/tang.go"
+	"github.com/flatcar/mantle/kola/cluster"
+	"github.com/flatcar/mantle/kola/register"
+	"github.com/flatcar/mantle/kola/tests/util"
+	"github.com/flatcar/mantle/platform"
+	"github.com/flatcar/mantle/platform/conf"
+	"github.com/flatcar/mantle/platform/machine/qemu"
+	"github.com/flatcar/mantle/platform/machine/unprivqemu"
+)
+
+const (
+	// In the Ignition config below, we create a partition labeled ROOT on a second disk.
+	// As advised in the Flatcar documentation, we then remove the ROOT label from the existing
+	// root partition, which is vda9 in the QEMU disk image.
+	IgnitionConfigRootTang = `{
+		"ignition": {
```

Comment:
Good point!

My thinking was that it would be best to first merge clevis support and after that open a PR to update the Butane spec, to avoid confusion. But once the Butane spec is updated, I'd be happy to bump the `butane` Go dependency in `mantle` and rewrite the config here. We can either hold off the merge of this PR until then, or I can open a second PR later. I don't have a preference either way; let me know what option you think is best.